app/controllers/followings_controller.rb

```diff
@@ -0,0 +1,23 @@
+class FollowingsController < ApplicationController
+
+  def create
+    @follower = current_user
+    @followee = User.find(params[:id])
+    if @followee != @follower
+      Following.create(follower: @follower, followee: @followee)
+
+      respond_to do |format|
+        format.html { redirect_to profile_path(@followee) }
+        format.js
+      end
+    end
+  end
```

Comment:
Oops I forgot that it would automatically look for the view file at the end of the action if not redirected. Will amend commit to redirect regardless. Thanks for the catch @gohdaniel15!